src/app/app.module.ts

```diff
@@ -147,6 +149,8 @@ import {AdminIssuerNewComponent} from './admin/admin-issuer-new/admin-issuer-new
     AdminCampaignAddTokensComponent,
     AdminCampaignDetailComponent,
     AdminIssuerNewComponent,
+    AdminIssuerEditCampaignVisibilityComponent,
+    AdminCampaignVisibilityCardComponent,
```

Comment:
maybe change from `AdminCampaignVisibilityCardComponent` to `AdminIssuerEditCampaignVisibilityCardComponent` ?